owlbot.py

```diff
@@ -72,10 +72,28 @@
     "import re\nimport shutil",
 )
 
+
 s.replace(
     ["noxfile.py"], "--cov=google", "--cov=sqlalchemy_bigquery",
 )
 
+
+s.replace(
+    ["noxfile.py"], 
+    "\+ SYSTEM_TEST_EXTRAS",
+    "",
+)
+
+
+s.replace(
```

Comment:
I always worry about brittleness with these kinds of replacements, but alas that's problem for future us.

Reply:
you and me both. 
i really disliked these `s.replace()` functions as I was writing them.